core/include/pbf/pbf.hpp

```diff
@@ -151,6 +152,14 @@ std::string message::string()
     return std::string(string, len);
 }
 
+std::pair<size_t, const char*> message::chunk()
+{
+    uint64_t len = varint();
+    value_type string = static_cast<value_type>(data_);
+    skipBytes(len);
+    return { len, string };
+}
+
```

Comment:
This method is used exactly once. I think we should avoid generalizing code until it's used at least twice, but since it's already written, meh :P